src/sequence_batch_scheduler.cc

```diff
@@ -546,6 +546,15 @@ SequenceBatchScheduler::Enqueue(std::unique_ptr<InferenceRequest>& irequest)
   const bool seq_end =
       ((irequest->Flags() & TRITONSERVER_REQUEST_FLAG_SEQUENCE_END) != 0);
 
+  // Check if the request is one of the in-flight sequence (not starting new
+  // sequence), currently we consider sequences in backlog are also in-flight
```

Comment:
"sequence). We consider sequences in backlog as also in-flight."

Not just for simplicity but also because we want that semantic. Any sequence that client has been able to start in the server we want to allow to finish.